src/integration-tests/bash/cleanup.sh

```diff
@@ -20,34 +20,21 @@
 #                   given lease on a failure.
 #
 #   SHARED_CLUSTER  Set this to true if you want cleanup to delete tiller
-#                   TBD tiller delete is disabled 
 #
 #   DELETE_FILES    Delete local test files, and launch a job to delete PV
 #                   hosted test files (default true).
 #
 #   FAST_DELETE     Set to "--grace-period=1 --timeout=1" to speedup
 #                   deletes and skip phase 2.
 #
-# Dry run option: 
-#
-#   To show what the script would do without actually doing
-#   any deletes pass "-dryrun" as the first parameter.
-#
 # --------------------
```

Comment:
Definitely looks like a bad merge or some other git issue.   It looks like your diff here is vs. develop instead of MII.    Develop has  the 'Dry run' stuff - it's the latest and newer than what's in MII.  Where-as the version you changed does not.

Reply:
Not sure why, it shouldn't have those now.

Reply:
The diff still looks bad.   'Dry run' should not have been removed.

Reply:
I am comparing with origin/model-in-image and I don't see Dry Run

Reply:
But the diff is showing dryRun - so something is definitely wrong.   This is somehow comparing to the develop branch version instead of the MII version.   Note that the develop branch version is newer.